app-misc/piper/piper-0.4.ebuild

```diff
@@ -18,13 +18,20 @@ IUSE=""
 REQUIRED_USE="${PYTHON_REQUIRED_USE}"
 
 BDEPEND="
+	${PYTHON_DEPS}
 	virtual/pkgconfig
 "
 RDEPEND="
 	${PYTHON_DEPS}
-	dev-libs/libratbag
-	$(python_gen_cond_dep 'dev-python/python-evdev[${PYTHON_MULTI_USEDEP}]')
+	>=dev-libs/libratbag-0.13
+	dev-python/lxml
+	dev-python/pycairo
+	gnome-base/librsvg[introspection]
+	x11-libs/gdk-pixbuf[introspection]
+	x11-libs/gtk+:3[introspection]
+	$(python_gen_cond_dep 'dev-libs/gobject-introspection[${PYTHON_SINGLE_USEDEP}]')
```

Comment:
`$(python_gen_cond_dep '` and `[${PYTHON_SINGLE_USEDEP}]')` should be dropped.
`giscanner` module is not used by either `dev-libs/libratbag` or `app-misc/piper`.
`giscanner` module is installed in custom directory (`/usr/lib64/gobject-introspection/giscanner`) instead of in `site-packages`, and is rather only for private usage by scripts from `dev-libs/gobject-introspection` (`/usr/bin/g-ir-annotation-tool`, `/usr/bin/g-ir-scanner`).